install_config/certificate_customization.adoc

```diff
@@ -97,4 +97,4 @@ the server to pick up the configuration changes.
 For the master API or web console, wildcard names are accepted.
 
 The provided named certificate can also be added using the 
-xref:../install_config/redeploying_certificates.adoc#redeploying-new-custom-ca[*_redeploy-certificates.yml_* playbook]. 
+xref:../install_config/redeploying_certificates.adoc#redeploying-all-certificates-using-the-current-product-title-and-etcd-ca[*_redeploy-certificates.yml_* playbook].  
```

Comment:
@mburke5678 - this link is not correct. The actual anchor is: redeploying-all-certificates-current-ca.

When gpei pointed you to the text, he was pointing you to the text on your branch. You should have double checked that the anchor text exists in the target file. Can you please follow up and fix?

Reply:
Fixed https://github.com/openshift/openshift-docs/pull/8351 
Thank you.